src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Dropdown.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '../../components/Icon';
+import dropdownStyles from './dropdown.scss';
+
+export default class Dropdown extends React.PureComponent {
```

Comment:
Maybe rename to something like `DropdownList`? `DropdownButtonList`? Simply naming it dropdown associated a different component for me.